CODEOWNERS

```diff
@@ -1,9 +1,9 @@
 # Products
 
-/products/rhel*/product.yml @ComplianceAsCode/red-hatters
-/products/ol*/product.yml @ComplianceAsCode/oracle-maintainers
-/products/sle*/product.yml @ComplianceAsCode/suse-maintainers
-/products/ubuntu*/product.yml @ComplianceAsCode/ubuntu-maintainers
+/tests/data/product_stability/rhel*.yml @ComplianceAsCode/red-hatters
+/tests/data/product_stability/ol*.yml @ComplianceAsCode/oracle-maintainers
+/tests/data/product_stability/sle*.yml @ComplianceAsCode/suse-maintainers
+/tests/data/product_stability/ubuntu*.yml @ComplianceAsCode/ubuntu-maintainers
```

Comment:
It is good to include these new files in CODEOWNERS, but doesn't seem necessary to remove the existing lines since the files can still be updated.

Reply:
@matejak , did you have an opportunity to check this?

Reply:
Sorry for not replying sooner, I think that keeping the previous files would be redundant. The new files reflect changes done to product yamls as well as product properties, so it doesn't make much sense to pay attention to one of those aspects more, as both can change the product's behavior.
In the not so distant future, we should aim to move more attributes from rules directly to product properties, so the importance of rule yamls will relatively decrease.

Reply:
Ok. I see. Fortunately this meantime I worked in some changes which impacted in the product stability tests. So, basically any change in product properties will generate errors in the product stability tests, naturally demanding updates in the `/tests/data/product_stability/` files you included in the CODEOWNERS. 

Basically you are considering redundant because a PR would not pass in CI tests without updating the `/tests/data/product_stability/` files in these cases. I was thinking in a scenario without a dependency of CI tests so any change in files related to products would require products maintainers to take a look.

Although I would prefer more explicitness of the files, this approach also works fine.
**It is clear for me now and I am fine as it is**.

Thanks